requirements.txt

```diff
@@ -1,9 +1,7 @@
--r penaltymodel_core/requirements.txt
-
--r penaltymodel_cache/requirements.txt
-
--r penaltymodel_maxgap/requirements.txt
-
--r penaltymodel_mip/requirements.txt
-
--r penaltymodel_lp/requirements.txt
+dimod==0.10.10
+homebase==1.0.1
+networkx==2.4
+numpy==1.19.1;python_version<"3.9"
```

Comment:
Now updated to 1.20?

Reply:
yes, I'll update the scipy version as well.